reader.c

```diff
@@ -151,6 +212,10 @@ int nsq_reader_connect_to_nsqd(struct NSQReader *rdr, const char *address, int p
     struct NSQDConnection *conn;
     int rc;
 
+    if (rdr->lookupd == NULL) {
+        rdr->address = strdup(address);
+        rdr->port = port;
+    }
```

Comment:
A standalone `nsqd` means a single `nsqd` node, doesn't it? If there is a `nsqd` cluster, it's better to use `nsqlookupd`, right? Or else, that would be a complicated situation. If a reader connect to many `nsqd` directly, it will never stop to retry connect to a quitted node.